com.github.reds.LogisimEvolution.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 REDS institute -->
+<component type="desktop-application">
+ <id>com.github.reds.LogisimEvolution</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-3.0</project_license>
+ <name>Logisim Evolution</name>
+ <summary>Digital logic designer and simulator</summary>
+ <developer_name>REDS institute</developer_name>
+ <url type="homepage">https://github.com/reds-heig/logisim-evolution</url>
+ <description>
+  <p>
+  Logisim is an educational tool for designing and simulating digital logic circuits.
+  It has been originally created by Dr. Carl Burch and actively developed until 2011.
+  After this date the author focused on other projects, and recently the development
+  has been officially stopped (see his message here).
+</p>
+  <p>
+  In the meantime, people from a group of swiss institutes (Haute École Spécialisée Bernoise,
+  Haute École du paysage, d'ingénierie et d'architecture de Genève,
+  and Haute École d'Ingénierie et de Gestion du Canton de Vaud) started developing
+  a version of Logisim that fitted their courses, integrating several tools -- for instance
+  a chronogram, the possibility to test the schematics directly on an electronic board,
+  TCL/TK consoles, ...
+</p>
+  <p>
+  We have decided to release this new Logisim version under the name logisim-evolution,
+  to highlight the large number of changes that occurred in these years,
+  and we actively seek the contribution of the community.
+</p>
+ </description>
+ <screenshots>
+  <screenshot type="default">
+   <image>http://www.cburch.com/logisim/shot-2.7.0.png</image>
+  </screenshot>
+ </screenshots>
+ <releases>
+   <release version="3.3.0" date="2020-01-09"/>
+ </releases>
+ <content_rating type="oars-1.1" />
+ </content_rating>
```

Comment:
Remove this line.

Reply:
Only line 41 right? I can't tell exactly what line when someone is reviewing.

Reply:
Thank you!